cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -633,6 +645,99 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    VisibilityAndAccessEditor = AbstractVisibilityAndAccessEditor.extend({
+        templateName: 'visibility-and-access-editor',
```

Comment:
This template is just the access part now, correct? I think that same renaming/cleanup still needs to be done.